### PR TITLE
feat(inoreader): support for new ui

### LINKED ISSUE #979: Support Inoreader's new UI!
### Is there an existing issue outlining your problem?

- [X] I am using the latest version of the provided userstyle.
- [X] I have searched the existing issues and they do not solve my problem.

### What userstyles are you seeing the problem on?

lbl:inoreader

### Describe your problem.

The new Inoreader UI rolled out to Beta users, and will be enforced to all users on July 1st.
[source](https://mastodon.social/@Inoreader/112598076579595851)

### Attach screenshots.

image

### What browser(s) are you seeing the problem on?

Chromium v125.0.6422.112

### Any additional comments?

_No response_

### PR DESCRIPTION
**Pull request by Thomas-Philippot**
## 🔧 What does this fix? 🔧

Support new inoreader UI (beta) 
Closes #979 

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/inoreader/catppuccin.user.css [hunk 1]
```diff
@@ -69,309 +69,485 @@
...
+    color: @blue !important;
+  }
 
-    #new_articles_overlay {
-      color: @accent-color;
-      background-color: @base;
-    }
+  .tabs-counter {
+    background-color: @blue !important;
+    outline: none !important;
```
Comment (somerand0mcat) [inline styles/inoreader/catppuccin.user.css:209]: we don't want an opinionated change

File: styles/inoreader/catppuccin.user.css [hunk 2]
```diff
@@ -69,309 +69,485 @@
...
+    color: @yellow;
+  }
 
-    div.article_magazine.article_current,
-    div.article_magazine.article_bulk_selected {
-      box-shadow: 0 0 0 1px @subtext0;
-    }
+  .bg-alt {
+    background-color: @yellow;
+    background: @yellow !important;
```
Comment (somerand0mcat) [inline styles/inoreader/catppuccin.user.css:187]: why are you using both ``background-color`` and ``background``?

File: styles/inoreader/catppuccin.user.css [hunk 3]
```diff
@@ -69,309 +69,485 @@
...
-      color: @mantle;
-    }
+  div.article_magazine.article_current,
+  div.article_magazine.article_bulk_selected {
+    box-shadow: 0 0 0 1px @subtext0;
+  }
+  .view_style_2 #reader_pane .ar.article_current_3way,
+  .view_style_2 #reader_pane .ar.article_expanded {
+    box-shadow: none;
+    border: 1px solid @text;
```
Comment (somerand0mcat) [inline styles/inoreader/catppuccin.user.css:344]: we dont want a opinionated change

**Review by somerand0mcat — CHANGES_REQUESTED** [review]

File: styles/inoreader/catppuccin.user.css (hunk 3 above)
Comment (isabelroses) [reply styles/inoreader/catppuccin.user.css:344]: to build on this. consider using `border-color` instead.

Comment (uncenter) [reply styles/inoreader/catppuccin.user.css:344]: > we dont want a opinionated change

The linter will catch issues like this. No need to comment on them Omar.

File: styles/inoreader/catppuccin.user.css (hunk 2 above)
Comment (uncenter) [reply styles/inoreader/catppuccin.user.css:187]: The linter catches this. Please don't comment on it Omar.

File: styles/inoreader/catppuccin.user.css (hunk 1 above)
Comment (uncenter) [reply styles/inoreader/catppuccin.user.css:209]: Again, the linter catches this.

**Review by isabelroses — APPROVED** [review]

**Review by somerand0mcat — APPROVED** [review]

**Comment by isabelroses** [comment]
In future @Thomas-Philippot you can merge your own PRs if they have been approved.

**Pull request merged**